### PR TITLE
Update microRTPS and ROS 2 docs

### PR DESCRIPTION
**Pull request by TSC21**
Brings some updates and cleanup to relevant parts of the the documentation for the install and usage of microRTPS and ROS2 with PX4, also following the updates in the latest big updates to the bridge UX and functionalities.

### CODE REVIEW

File: en/ros/ros2_offboard_control.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ If you are operating on a real vehicle be sure to have a way of gaining back man
 :::
 
 :::warning
-ROS 2 interaction with PX4 through the [*microRTPS* bridge](../middleware/micrortps.md) requires that the user understands how the PX4 internals work!
+ROS 2 interaction with PX4, done through the [*microRTPS* bridge](../ros/ros2_comm.md), requires that the user understands how the PX4 internals work!
```
Comment (hamishwillee) [inline en/ros/ros2_offboard_control.md:9]: Probably is change is not helpful. But it doesn't break anything.

File: en/simulation/multi_vehicle_simulation_gazebo.md [hunk 2]
```diff
@@ -57,11 +57,11 @@ The `MAV_SYS_ID` and various UDP ports are allocated in the SITL rcS: [init.d-po
 
 To simulate multiple vehicles based on RTPS/DDS in Gazebo, use the `gazebo_sitl_multiple_run.sh` command in the terminal with the `-t px4_sitl_rtps` option from the root of the *PX4-Autopilot* tree (as described above).
 Here we will use the `-t px4_sitl_rtps` option, which sets that we will use RTPS for communicating with PX4 rather than the MAVLink Simulation API.
-This will build and run the `iris_rtps` model (the only model that is currently implemented for use with RTPS).
+This will build and runs the `iris` model (can be changed by the `-m` parameter), and **by default also starts the microRTPS client**.
```
Comment (hamishwillee) [inline en/simulation/multi_vehicle_simulation_gazebo.md:60]: run, not runs.

File: en/dev_setup/fast-dds-installation.md [hunk 3]
```diff
@@ -120,54 +126,15 @@ The latest binary release of *eProsima Fast DDS* can be downloaded from the [com
 
 Documentation on how to do this can be found here: [Installation from Binaries on Linux](https://fast-dds.docs.eprosima.com/en/latest/installation/binaries/binaries_linux.html) and [Installation from Binaries on Windows](https://fast-dds.docs.eprosima.com/en/latest/installation/binaries/binaries_windows.html) (*eProsima Fast DDS* official documentation)
 
-
```
Comment (hamishwillee) [inline en/dev_setup/fast-dds-installation.md:123]: @TSC21 Windows not supported?

**Comment by hamishwillee** [comment]
@TSC21 Much better :-). I have subedited the changed sections and will merge. 

I assume Windows no longer supported, since that was removed.

**Comment by TSC21** [comment]
> @TSC21 Much better :-). I have subedited the changed sections and will merge.
> 
> I assume Windows no longer supported, since that was removed.

Windows is supported by FastDDS and ROS2  But I wanna make sure we keep things focused on Ubuntu for now. Otherwise it becomes difficult to maintain as I am not using Windows for development and so then I can't provide insights if someone runs into a problem. Maybe later we can readd it and add a Windows section for our ROS2 docs as well.

**Comment by TSC21** [comment]
Thanks for the merging and subedits.

**Comment by hamishwillee** [comment]
@TSC21 

Re "Windows is supported by FastDDS and ROS2 But I ..."

Absolutely fair enough. You might want to consider adding a note to this effect to the docs. I.e. you could have a windows section and simply state "Windows is supported by FastDDS and ROS2. However the PX4 team do not support Windows ROS builds at this time". That closes down potential questions. Up to you.

**Pull request merged**